plugin/trino-hive/src/main/java/io/trino/plugin/hive/ViewReaderUtil.java

```diff
@@ -183,7 +183,7 @@ public static boolean isTrinoMaterializedView(String tableType, Map<String, Stri
         // https://github.com/trinodb/trino/blame/ff4a1e31fb9cb49f1b960abfc16ad469e7126a64/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java#L898
         return tableType.equals(VIRTUAL_VIEW.name()) &&
                 "true".equals(tableParameters.get(PRESTO_VIEW_FLAG)) &&
-                tableParameters.get(TABLE_COMMENT).equalsIgnoreCase(ICEBERG_MATERIALIZED_VIEW_COMMENT);
+                ICEBERG_MATERIALIZED_VIEW_COMMENT.equalsIgnoreCase(tableParameters.get(TABLE_COMMENT));
```

Comment:
Follow-up is fine. Is it possible to add a test? Did you get NPE with Glue environment?